rules/falco_rules.yaml

```diff
@@ -1902,6 +1902,32 @@
   priority: INFO
   tags: [container, cis, mitre_privilege_escalation, mitre_lateral_movement]
 
+# These capabilities were used in the past to escape from containers
+- macro: excessively_capable_container
+  condition: >
+    (thread.cap_permitted contains CAP_SYS_ADMIN
+    or thread.cap_permitted contains CAP_SYS_MODULE
+    or thread.cap_permitted contains CAP_SYS_RAWIO
+    or thread.cap_permitted contains CAP_SYS_CHROOT
+    or thread.cap_permitted contains CAP_SYS_PTRACE
+    or thread.cap_permitted contains CAP_SYS_BOOT
+    or thread.cap_permitted contains CAP_SYSLOG
+    or thread.cap_permitted contains CAP_DAC_READ_SEARCH
+    or thread.cap_permitted contains CAP_NET_ADMIN
+    or thread.cap_permitted contains CAP_NET_RAW)
```

Comment:
`CAP_NET_RAW` is a default capability, it's going to trigger the rule `Launch Excessively Capable Container` for any container comes up.

Reply:
You are right, I wasn't seeing it because I was testing launching an ubuntu container and the rule shell in container was obscuring this one. Thanks! I am modifying it now